stable/vault-operator/values.yaml

```diff
@@ -4,7 +4,7 @@ replicaCount: 1
 image:
   repository: quay.io/coreos/vault-operator
   tag: 0.1.9
-  pullPolicy: IfNotPresent
+  pullPolicy: Always
```

Comment:
Why the force pull by default?

Reply:
This is a value that helm takes during deployment. According to the README.md this value is supposed to be pull by default, but in the values.yaml it is not. I was relying on this default parameter during a major deployment and it caused a lot of trouble because the README.md said the default was "Always" but was actually set to "IfNotPresent" causing my deployment to use the cached version of the image. You can see this in the Parameter Configuration section of the README.md.

Reply:
My suggestion is to go the other way; update the README to match the values. Most charts default to `IfNotPresent`, and since we aren't changing tags frequently, the caching is (arguably) a better approach.

Reply:
As the vault-operator project itself, has been dead since August 2018, I suggest we make the default "Always" as engineers will almost always be using this chart with their own custom image like I am doing right now. The project was left in beta release and has a lot of bugs, so this chart should account for engineers using their own custom images especially while testing. An "Always" would be helpful as engineers may not be changing tags as frequently as CoreOS does, given that this project is dead from their end. I don't disagree with you, but I was an end-user of this chart and I believe setting an "Always" default would be helpful for it in the longer run.

Reply:
If you still think otherwise, then I'll go ahead and make the change to update the README instead. Let me know.

Reply:
Ah, I see - if there is ever a new champion for VO, we can revisit this.

Reply:
I have already forked it and am trying to get the fork more focus from the community as this is a widely used operator so it should be continuously developed on and improved.